flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobMaster.java

```diff
@@ -343,10 +346,10 @@ private SchedulerNG createScheduler(
                 slotPoolServiceSchedulerFactory.createScheduler(
                         log,
                         jobGraph,
-                        scheduledExecutorService,
+                        ioExecutor,
```

Comment:
Let's also update the parameter names in `SlotPoolServiceSchedulerFactory.createScheduler` and its implementations.